src/main/java/capstone/mfslbackend/controller/TeamController.java

```diff
@@ -23,8 +24,10 @@ public TeamController(TeamService teamService) {
         this.teamService = teamService;
     }
 
-    @GetMapping("{teamId}")
-    public ResponseEntity<Team> getTeam(@PathVariable long teamId) {
+
+    @GetMapping("/team")
+    @Secured({"LEAGUE_MEMBER", "LEAGUE_ADMIN"})
```

Comment:
@linamoufarreh @seb8stien1 This tag allows you to set which user roles can call that controller method.

For example, if we want certain controller methods to be only called by admin, we would just have `"LEAGUE_ADMIN"` in this tag.